Annex60/Controls/Continuous/PIDHysteresis.mo

```diff
@@ -52,28 +52,32 @@ model PIDHysteresis
   parameter Real y_start=0 "Initial value of output"
     annotation (Dialog(group="Initialization"));
 
+  parameter Boolean strict=true "= true, if strict limits with noEvent(..)"
+    annotation (Evaluate=true, choices(checkBox=true), Dialog(tab="Advanced"));
+
   LimPID PID(
-    controllerType=controllerType,
-    k=k,
-    Ti=Ti,
-    yMax=yMax,
-    yMin=yMin,
-    wp=wp,
-    wd=wd,
-    Ni=Ni,
-    Nd=Nd,
-    initType=initType,
-    limitsAtInit=limitsAtInit,
-    xi_start=xi_start,
-    xd_start=xd_start,
-    y_start=y_start,
-    Td=Td,
-    reverseAction=reverseAction) "Controller for room temperature"
+    final controllerType=controllerType,
+    final k=k,
+    final Ti=Ti,
+    final yMax=yMax,
+    final yMin=yMin,
+    final wp=wp,
+    final wd=wd,
+    final Ni=Ni,
+    final Nd=Nd,
+    final initType=initType,
+    final limitsAtInit=limitsAtInit,
+    final xi_start=xi_start,
+    final xd_start=xd_start,
+    final y_start=y_start,
+    final Td=Td,
+    final reverseAction=reverseAction,
+    final strict=strict) "Controller for room temperature"
     annotation (Placement(transformation(extent={{-30,-2},{-10,18}})));
   Modelica.Blocks.Logical.Hysteresis hys(
-    pre_y_start=pre_y_start,
-    uLow=eOff,
-    uHigh=eOn) "Hysteresis element to switch controller on and off"
+    final pre_y_start=pre_y_start,
+    final uLow=eOff,
+    final uHigh=eOn) "Hysteresis element to switch controller on and off"
     annotation (Placement(transformation(extent={{-30,50},{-10,70}})));
   Modelica.Blocks.Logical.Switch swi
     annotation (Placement(transformation(extent={{60,-10},{80,10}})));
```

Comment:
In PIDHysteresisTimer, you set this to `zer(final k=0)`, @mwetter . Is there a reason why it is different for this model here?